.eslintrc.cjs

```diff
@@ -25,5 +25,12 @@ module.exports = {
                 'import/no-unresolved': 'off',
             },
         },
+        {
+            files: ['packages/safe-ds-editor/src/**'],
+            rules: {
+                'import/no-default-export': 'off',
+                'no-console': 'off',
+            },
+        },
```

Comment:
I don't see a reason to disable these rules completely for this folder.

- Default exports are rarely needed, and using named exports throughout leads to more consistent code and a more consistent API. Moreover, we provide names for the exported elements, which already indicate their purpose. Default exports are unnamed, so users must always refer to documentation or the module that exports them.
- Console statements are very easy to forget in the code. You are better off disabling the error for the few lines where you need them by adding a comment directly above the affected lines (https://eslint.org/docs/latest/use/configure/rules#disabling-rules).